### DISCUSSION TITLE
Welcome to burstcamp Discussions!

### DISCUSSION THREAD

**Discussion by sseungmn** [boostcampwm-2022/iOS09-burstcamp#169] [Announcements]
## 👋 Welcome!
  이 곳은 burstcamp의 Discussion입니다.
  팀원들끼리 의견을 나누기 위해 만들어진 곳이지만, 누구나 의견을 제시할 수 있습니다.
  * 프로젝트에 대해 궁금한 것을 물어볼 수 있습니다.
  * 여러가지 문제에 대한 다양한 아이디어를 나눌 수 있습니다.
  * 부스트캠프가 끝난 이후에도 지속적인 교류를 지양합니다. 
    언제든, 어떤 이야기든 저희에게 들려주세요.

**Comment by never-better** [comment]
오픈 ㅊㅊ

**Comment by sseungmn** [comment]
신장개업 **버캠 디스커션**

**Comment by never-better** [comment]
슬랙에 알림 뜨나 확인용

**Comment by chansooo** [comment]
ㅊㅊㅊ
